examples/robolectricExample/build.gradle

```diff
@@ -0,0 +1,76 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'de.undercouch:gradle-download-task:2.0.0'
+    }
+}
+
+apply plugin: 'de.undercouch.download'
+apply plugin: 'android-sdk-manager'
+apply plugin: 'com.android.application'
+apply plugin: 'android-command'
+apply plugin: 'com.neenbedankt.android-apt'
+apply plugin: 'realm-android'
+
+def currentVersion = file("${rootDir}/../version.txt").text.trim()
+
+android {
+    compileSdkVersion rootProject.sdkVersion
+    buildToolsVersion rootProject.buildTools
+
+    defaultConfig {
+        applicationId 'io.realm.examples.robolectric'
+        targetSdkVersion 21
+        minSdkVersion 15
+        versionCode 1
+        versionName "1.0"
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+        }
+    }
+
+    command {
+        events 2000
+    }
+}
+
+dependencies {
+    compile fileTree(dir: 'libs', include: ['*.jar'])
+    compile 'com.android.support:appcompat-v7:23.1.0'
+    compile 'com.android.support:design:23.1.0'
+    testCompile 'io.reactivex:rxjava:1.1.0'
+    testCompile 'junit:junit:4.12'
+    testCompile "org.robolectric:robolectric:3.0"
+
+    testApt "io.realm:realm-annotations-processor:${currentVersion}"
+}
+
+task downloadRealmRobolectric {
```

Comment:
https://github.com/realm/realm-java/tree/master/gradle-plugin, it will require people using the new Realm Gradle plugin, but I think that is acceptable.